app/src/main/java/com/pvp/app/ui/screen/settings/Settings.kt

```diff
@@ -78,6 +85,35 @@ fun CategoryRow(
     )
 }
 
+@Composable
+fun ReminderSetting() {
+    Row {
+        Text(
+            text = "Reminder time",
+            style = MaterialTheme.typography.titleLarge,
+        )
+    }
+
+    Text(
+        modifier = Modifier.padding(vertical = 8.dp),
+        text = "Selected (minutes): $reminderTimeMinutes",
+    )
+
+    Slider(
```

Comment:
Maybe our own picker object from `../ui/common/Pickers.kt` would be handy here. Do not use `NumberPicker`, it is a redundant abstraction and I am going to dispose it soon.

I think we could and should use pickers in most places, where user is supposed to pick a value from a _range_.